### DISCUSSION TITLE
Synthesis reports

### DISCUSSION THREAD

**Discussion by suppamax** [siliconcompiler/siliconcompiler#1126] [Help (Q&A)] [answered]
Hi,

from the synthesis report I can easily find the area estimation.
Is there a way to get information about timing closure and power consumption?

**Comment by nmoroze** [comment, marked as answer]
I don't think the synthesis reports have any information related to timing closure or power consumption. You can find timing and power related information at the end of the logs output by each OpenROAD step later on in the flow.
